buddyboss/buddyboss-platform
# PROD-694 Performance improved - activity loading time reduced when multiple attached document

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3137 .

### How to test the changes in this Pull Request:

1. Create an activity item and add multiple( 10 ) image/pdf as document.
2. Make sure you have symlink is disabled
3. Go to the news feed page
4. See it is loading fast.

### Proof Screenshots or Video

https://www.loom.com/share/ea9c7664206e4d71845703c77ba539dc

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
